src/uct/ib/mlx5/gdaki/gdaki.c

```diff
@@ -126,19 +143,19 @@ uct_rc_gdaki_umem_reg(const uct_ib_md_t *md, struct ibv_context *ibv_context,
                       void *address, size_t length)
 {
     struct mlx5dv_devx_umem_in umem_in = {};
-    uct_cuda_copy_md_dmabuf_t dmabuf;
+    uct_cuda_copy_md_dmabuf_t dmabuf UCS_V_UNUSED;
 
     umem_in.addr        = address;
     umem_in.size        = length;
     umem_in.access      = IBV_ACCESS_LOCAL_WRITE;
     umem_in.pgsz_bitmap = UINT64_MAX & ~(ucs_get_page_size() - 1);
+#if HAVE_DECL_MLX5DV_UMEM_MASK_DMABUF
```

Comment:
Do we really need compile check also here?

Reply:
without it `umem_in.dmabuf_fd` and `MLX5DV_UMEM_MASK_DMABUF` would yield unknown identifier error